vm/address/src/network.rs

```diff
@@ -17,6 +19,17 @@ impl From<Network> for fvm_shared::address::Network {
     }
 }
 
+impl FromStr for Network {
+    type Err = ();
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "mainnet" => Ok(Network::Mainnet),
+            _ => Ok(Network::Testnet),
```

Comment:
Not sure about this, `"cthulhu"` shouldn't default to testnet - I'd expect an unrecognized network panic. Let's be pickier!

Reply:
I think it should, all networks that don't are `mainnet` are `testnet`.

Reply:
Done.